src/haddock/modules/analysis/caprieval/capri.py

```diff
@@ -563,13 +567,15 @@ def get_atoms(pdb_list):
             pdb = pdb.rel_path
         with open(pdb) as fh:
             for line in fh.readlines():
-                if line.startswith("ATOM"):
+                if line.startswith(("ATOM", "HETATM")):
                     resname = line[17:20].strip()
                     atom_name = line[12:16].strip()
                     element = line[76:78].strip()
-                    if (resname not in PROT_RES
+                    if (
+                            resname not in PROT_RES
                             and resname not in DNA_RES
-                            and resname not in IGNORE_RES):
```

Comment:
What is this ignore _res list? If to be ignored, shouldn't it be rather: `and rename in IGNORE_RES`?

Reply:
These are defined in the beginning of the file: https://github.com/haddocking/haddock3/blob/b92d2c1ff104f81684ed9eb2cff393ec8aa90d46/src/haddock/modules/analysis/caprieval/capri.py#L23-L69

Note to self: update this in #144

Reply:
made it more clear with:
```python
and resname not in RES_TO_BE_IGNORED
```